final_installation/terminal_server/workshop_pollen_metabarcoding/install_software.sh

```diff
@@ -10,17 +16,11 @@ pip install qiime
 #figtree
 apt install --yes --force-yes figtree
 
-#wget www.drive5.com/uclust/uclustq1.2.22_i86linux32
-#mv uclustq1.2.22_i86linux32 /usr/local/bin/uclust
-#chmod +x /usr/local/bin/uclust
-#raxml
-#git clone https://github.com/stamatak/standard-RAxML.git
-#cd standard-RAxML
-#make -f Makefile.PTHREADS.gcc
-#rm *.o
-#cd ..
-#chmod 755 standard-RAxML/raxmlHPC-PTHREADS
-#ln -fs $PWD/standard-RAxML/raxmlHPC-PTHREADS /usr/local/bin/raxml
-#Usearch
-#chmod 755 $PWD/usearch8.0.1623_i86linux32
-#ln -fs $PWD/usearch8.0.1623_i86linux32 /usr/local/bin/usearch
+#ea-utils
+apt install --assume-yes --no-install-recommends ea-utils
+
+# install optional software (SeqFilter/usearch) via special package
+cd /tmp
+git clone git@132.187.22.105:alex-summerschool-2016/opt-software
+cd opt-software
+dpkg -i summerschool-opt-software_1.0.0_amd64.deb
```

Comment:
I tested the installation of this deb package in a fresh ubuntu docker container. Both usearch and SeqFilter were properly installed.